src/main/java/de/tum/in/test/api/ast/data/UnwantedNodesInformation.java

```diff
@@ -0,0 +1,47 @@
+package de.tum.in.test.api.ast.data;
+
+import com.github.javaparser.ast.CompilationUnit;
+import com.github.javaparser.ast.Node;
+import com.github.javaparser.utils.Pair;
+import de.tum.in.test.api.ast.manager.NodeManager;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Stores all unwanted nodes of an abstract syntax tree of a Java-file
+ */
+public class UnwantedNodesInformation {
+    /**
+     * List of node-name / node-position pairs of unwanted nodes
+     */
```

Comment:
[Implemented part of the feedback](https://github.com/ls1intum/Ares/pull/248/commits/7ed9fe2ac7f61d955c71ff99f56e550f8be89531): Removed the JavaDoc in those places where it might be repetitive.